style.css

```diff
@@ -301,10 +475,11 @@
 
 .item2 {
   display: flex;
+  width: 100%;
   background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 700.94%), url(/Assets/images/backimage.png);
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement because as I can see your card background gradient color is not matching with the original **Figma** design card, which makes your design `inconsistent` and looks unprofessional. Kindly fix the colour issue have a look at the below-attached design comparison table and also please add the interactions like hover effects as given in the **Figma** design.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

![image](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/66476329/c3c1364d-4fb5-43ff-be2f-05aa30062fb2)


### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-17 13-21-41](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/66476329/dbd2f155-4e4a-4967-9344-b44a0d3320e2)

</td>
<td>

![image](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/66476329/6162b2ac-17cd-48b9-b04c-4c870e0a1612)

</td>
</tr>
</table>


### Design hover effect demo (❗❗ IMPORTANT ❗❗)

[Screencast from 17-06-23 01:25:19 PM IST.webm](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/66476329/db536e5f-e612-4afc-be4f-bcc2fcc0dd13)